packages/components/pie-toast-provider/package.json

```diff
@@ -41,6 +41,7 @@
     "cem-plugin-module-file-extensions": "0.0.5"
   },
   "dependencies": {
+    "@justeattakeaway/pie-toast": "0.5.0",
```

Comment:
I think this should also include button and tag if they're being used in the stories. However this does add a dependency when building, do we need to use our components for this?

Reply:
button and tag aren't part of the component code itself so I don't think we should include storybook-related dependencies to the component dependency tree. at least we didn't do it for all other components. 

https://github.com/justeattakeaway/pie/blob/main/packages/components/pie-toast-provider/src/index.ts#L15

Reply:
I think you're right actually, given that Storybook already depends on all the components. I was thinking of some components which have extra dependencies because of other components being imported in test files